config/locales/service_providers/fr.yml

```diff
@@ -18,13 +20,18 @@ fr:
       body_html: Si vous avez déjà un profil USAJOBS, veuillez utiliser l'adresse
         e-mail principale ou secondaire que vous avez utilisée pour USAJOBS pour %{link}.
       create_account_link: créer votre nouveau compte login.gov
+      create_account_page:
+        body: ''
     learn_more: En savoir plus.
     sam:
       header: Êtes-vous venu(e) de SAM?
       account_page:
-        body: Si vous avez déjà un profil SAM, veuillez utiliser l'adresse e-mail
-          principale ou secondaire que vous avez utilisée pour SAM pour créer votre
-          nouveau compte login.gov.
+        body: Votre ancien nom d'utilisateur et mot de passe SAM ne fonctionnera pas.
+          S'il vous plaît créer un login.gov compte en utilisant la même adresse e-mail
+          que vous utilisez pour SAM.
       body_html: Si vous avez déjà un profil SAM, veuillez utiliser l'adresse e-mail
         principale ou secondaire que vous avez utilisée pour SAM pour %{link}.
       create_account_link: créer votre nouveau compte login.gov
+      create_account_page:
+        body: S'il vous plaît créer un login.gov compte en utilisant la même adresse
+          e-mail que vous utilisez pour SAM.
```

Comment:
Please use this text: `Veuillez créer un compte login.gov avec la même adresse e-mail que vous avez utilisée pour SAM.`